app/assets/javascripts/angular/controllers/graph_ctrl.js

```diff
@@ -135,7 +135,6 @@ angular.module("Prometheus.controllers").controller('GraphCtrl',
       debounce = $timeout(function() {
         refreshFn($scope.graph.endTime, $scope.graph.range, step).then(function(data) {
           scope.$broadcast('redrawGraphs', data);
-          AnnotationRefresher(scope.graph, scope);
```

Comment:
Looks like you can remove `AnnotationRefresher` from the module dependencies now.